service/key/key.go

```diff
@@ -18,6 +19,19 @@ func ClusterID(customObject awstpr.CustomObject) string {
 	return customObject.Spec.Cluster.Cluster.ID
 }
 
+func ClusterVersion(customObject awstpr.CustomObject) string {
+	return customObject.Spec.Cluster.Version
+}
+
+func HasClusterVersion(customObject awstpr.CustomObject) bool {
```

Comment:
This key helper checks if a valid version is provided. This will be used whenever we need conditional logic for the security changes.